readthedocs/projects/templatetags/builds_tags.py

```diff
@@ -11,6 +15,4 @@ def get_docs_url(build):
     version = build.version
     private = version.privacy_level == PRIVATE
     return version.project.get_docs_url(
-        version_slug=version.slug,
-        private=private
-    )
+        version_slug=version.slug, private=private)
```

Comment:
I really liked it in the other way :disappointed:

Reply:
You just need to add a trailing comma in the `private` line.